_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Date Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to filter a list of events, or a report, by date ranges, or choose a single date for an event as they fill out a form.
+preview-image: preview-images/date-picker.svg
+status: unknown
+resource: true
+last-modified: 2017-10-22
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+Date picker is often used for applying date range filters to lists of events, logs, or even for filtering monitoring reports.  It can also be used for advanced filtering in areas such as ticketing and billing.  The picker may be used to specify dates or ranges in the process of submitting a form for use cases other than filtering as well.
+
+### Best practice
+ * Provide the range selection variation when a user needs to select both a start and an end date. 
+ * Pair the date picker pattern with data sets that extend across a time period to allow users to select the currently viewed data.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-hero-image.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+## Specifications
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Default layout
+The date picker component is composed of multiple buttons with four primary functions.
+
+ * Move the element to the next month
+ * Move the element to the previous month
+ * Selecting a specific date
+ * Move between the month selector and date selector states.
+
+When the Date picker is triggered, it should default to the current month and year
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-specifications.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Button states
+Text and icons should always be centered vertically and horizontally.
```

Comment:
Center text and icons vertically and horizontally.

Reply:
👍